pkg/controller/build/build_controller.go

```diff
@@ -459,6 +467,20 @@ func (r *ReconcileBuild) validateBuildRunOwnerReferences(ctx context.Context, b
 	return nil
 }
 
+// validateSourceUrl returns error message if remote repository doesn't exist
+func (r *ReconcileBuild) validateSourceUrl(ctx context.Context, b *build.Build, sourceUrl string) error {
```

Comment:
@qu1queee I don't think we need to verify source url is empty. Because [we don't allow sourceURL is empty](https://github.com/shipwright-io/build/blob/master/pkg/apis/build/v1alpha1/gitsource.go#L16), this case won't happen.

Reply:
:)